[2025-06-13T03:14:41.194+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-13T03:14:41.215+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: payparser_pipeline_dag.process_new_images manual__2025-06-13T03:14:38.729757+00:00 [queued]>
[2025-06-13T03:14:41.224+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: payparser_pipeline_dag.process_new_images manual__2025-06-13T03:14:38.729757+00:00 [queued]>
[2025-06-13T03:14:41.225+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-06-13T03:14:41.240+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): process_new_images> on 2025-06-13 03:14:38.729757+00:00
[2025-06-13T03:14:41.246+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=84) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-13T03:14:41.248+0000] {standard_task_runner.py:72} INFO - Started process 85 to run task
[2025-06-13T03:14:41.248+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'payparser_pipeline_dag', 'process_new_images', 'manual__2025-06-13T03:14:38.729757+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/payparser_dag.py', '--cfg-path', '/tmp/tmporxguoo5']
[2025-06-13T03:14:41.250+0000] {standard_task_runner.py:105} INFO - Job 75: Subtask process_new_images
[2025-06-13T03:14:41.307+0000] {task_command.py:467} INFO - Running <TaskInstance: payparser_pipeline_dag.process_new_images manual__2025-06-13T03:14:38.729757+00:00 [running]> on host 9b9192f456bf
[2025-06-13T03:14:41.394+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='zain' AIRFLOW_CTX_DAG_ID='payparser_pipeline_dag' AIRFLOW_CTX_TASK_ID='process_new_images' AIRFLOW_CTX_EXECUTION_DATE='2025-06-13T03:14:38.729757+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-13T03:14:38.729757+00:00'
[2025-06-13T03:14:41.395+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-06-13T03:14:41.420+0000] {logging_mixin.py:190} INFO - ⏳ Processing image: /opt/***/shared/downloads/WhatsApp Image 2025-06-13 at 01.34.27_5af07ffa.jpg
[2025-06-13T03:14:43.227+0000] {logging_mixin.py:190} INFO - Database error: unable to open database file
[2025-06-13T03:14:43.227+0000] {logging_mixin.py:190} INFO - ❌ Error processing WhatsApp Image 2025-06-13 at 01.34.27_5af07ffa.jpg: cannot access local variable 'conn' where it is not associated with a value
[2025-06-13T03:14:43.231+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-06-13T03:14:43.240+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-06-13T03:14:43.241+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=payparser_pipeline_dag, task_id=process_new_images, run_id=manual__2025-06-13T03:14:38.729757+00:00, execution_date=20250613T031438, start_date=20250613T031441, end_date=20250613T031443
[2025-06-13T03:14:43.311+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-13T03:14:43.325+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-13T03:14:43.328+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
